drivers/hmis/app/models/hmis/hud/project.rb

```diff
@@ -249,5 +264,9 @@ def occurrence_point_form_instances
     end.compact
   end
 
+  def uniq_coc_codes
+    @uniq_coc_codes ||= project_cocs.pluck(:CoCCode).uniq.compact_blank
```

Comment:
could the order matter here?
```suggestion
    @uniq_coc_codes ||= project_cocs.pluck(:CoCCode).uniq.compact_blank.sort
```